### ISSUE TITLE
Dart2 Support

### ISSUE THREAD

**Issue by brianegan** [brianegan/redux_logging#2]
Does it work!? Who knows! Apparently there might be an issue...

https://github.com/brianegan/flutter_redux/issues/28

**Comment by lasseschmitt** [comment]
The issue happens by calling the `formatter`
```
  void call(Store<State> store, action, NextDispatcher next) {
    next(action);

    logger.log(level, formatter(store.state, action, new DateTime.now()));
  }
```
https://github.com/brianegan/redux_logging/blob/master/lib/redux_logging.dart#L123

For me it feels like a Dart2 issue..

A quick fix to make it at least working again would be to write the formatted messaged directly 

```
  @override
  void call(Store<State> store, action, NextDispatcher next) {
    next(action);
    logger.log(level, "{Action: $action, State: ${store.state}, ts: ${new DateTime.now()}}");
  }
```

**Comment by brianegan** [comment]
@DrivEddy Published 0.2.0, should be good to go with Dart 2!
